test/active_model_serializers/register_jsonapi_renderer_test_isolated.rb

```diff
@@ -113,16 +109,21 @@ def test_jsonapi_parser_registered
     def test_jsonapi_renderer_registered
       expected = {
         'data' => {
-          'attributes' => {
-            'name' => 'Johnny Rico'
-          },
-          'type' => 'users'
+          'id' => 'author',
+          'type' => 'authors',
+          'attributes' => { 'name' => 'Johnny Rico' },
+          'relationships' => {
+            'posts' => { 'data' => nil },
+            'roles' => { 'data' => nil },
+            'bio' => { 'data' => nil }
+          }
         }
       }
+
       payload = '{"data": {"attributes": {"name": "Johnny Rico"}, "type": "authors"}}'
```

Comment:
I guess we should include a uuid here. how about `"id": ""36c9c04e-86b1-4636-a5b0-8616672d1765"` will make the expectation simpler. JSON API recommends that if a client wants to create an id on post, to use a uuid.

You've been working on this PR a while now, and the main goal of it is done. Thanks! I can do some of the other things I'd like in a followup :)